invenio_checks/templates/semantic-ui/invenio_checks/requests/details.html

```diff
@@ -0,0 +1,58 @@
+{# -*- coding: utf-8 -*-
+
+This file is part of Invenio.
+Copyright (C) 2025 CERN.
+
+Invenio is free software; you can redistribute it and/or modify it
+under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+{% import "invenio_checks/requests/severity_level_icons.html" as sli %}
+
+<div class="ui column stackable divided grid">
+  <div class="three wide column pr-0">
+
+    <h5 class="ui header text-muted">Checks</h5>
+    <div class="ui fluid secondary vertical menu ml-0 mr-0">
+      <a class="item active"><i class="{% include 'invenio_checks/requests/overall_severity_level.html' %}"></i> {{ _("Metadata check") }}</a>
+    </div>
+  </div>
+  <div class="twelve wide column">
+
+    <h5 class="ui header text-muted rel-pl-2 rel-pb-1">Rules</h5>
+
+    <div class="ui very relaxed list rel-pl-2">
+    {% for rule_result in checks.result.rule_results %}
+      {#
+        Get the icon class for the rule result of one rule result.
+        This code depends on 2 fields of each `rule_results` : `success` and `level`.
+        If the boolean `success` field is true, it means that the check was successful.
+        Otherwise, it means that the check failed, and we display the severity `level` of the check.
+      #}
+      {% set ns = namespace() %}
+      {%if rule_result.success %}
+        {% set ns.rule_severity_level = "success" %}
+      {% elif rule_result.level == "info" %}
+        {% set ns.rule_severity_level = "warning" %}
+      {% elif rule_result.level == "error" %}
+        {% set ns.rule_severity_level = "error" %}
+      {% else %}
+        {% set ns.rule_severity_level = "unknown" %}
```

Comment:
why not use it directly?
```suggestion
        {% set ns.rule_severity_level = rule_result.level %}
```

Reply:
It's not super obvious, this is why I'm explaining it in the comment above:

>         This code depends on 2 fields of each `rule_results` : `success` and `level`.
>         If the boolean `success` field is true, it means that the check was successful.
>         Otherwise, it means that the check failed, and we display the severity `level` of the check.